src/app/pages/home/home.controller.js

```diff
@@ -1,41 +1,26 @@
 'use strict';
 
-function MainController($scope, $filter, $rootScope, syncDataService,$state) {
-  'ngInject';
+export default class HomeController {
+  constructor(workWithDate, syncDataService, $state, $scope) {
+    'ngInject';
 
-  if (!$rootScope.currentUserId) {
-    $state.go('sign-up');
-  } else {
-    $scope.currentUserDeals = syncDataService.getDealsFromFirebase();
-  }
+    this.state = $state;
```

Comment:
for what do you need this variable?

Reply:
I don't know why Alexandra added it to home page, maybe she planned to use it in the future. But as I can see we don't need $state at this time and I just removed it from controller.